deploy/kubeturbo-operator/certified-operator-automation-1.py

```diff
@@ -0,0 +1,178 @@
+import os
```

Comment:
is there any reason to have -1 in filename ?

Reply:
It was a typo, will address it along other comments